### PR TITLE
Fix dbg_bps tests that always succeeded, regardless of output

### PR DESCRIPTION
**Pull request by yossizap**

### CODE REVIEW

File: test/new/db/archos/linux-x64/dbg_bps [hunk 1]
```diff
@@ -197,23 +197,27 @@ EOF
 RUN
 
 NAME='valid db'
-FILE=../bins/elf/analysis/x86-helloworld-gcc
+FILE=../bins/elf/analysis/x64-loop
```
Comment (radare) [inline test/new/db/archos/linux-x64/dbg_bps]: Why use a different bin?

Comment (yossizap) [reply test/new/db/archos/linux-x64/dbg_bps]: Hello world printed
```
valid
+Hello world!
```

When it was ran

File: test/new/db/archos/linux-x64/dbg_bps [hunk 2]
```diff
@@ -197,23 +197,27 @@ EOF
...
 NAME='invalid db'
-FILE=../bins/elf/analysis/x86-helloworld-gcc
+FILE=../bins/elf/analysis/x64-loop
 ARGS=-d
 CMDS=<<EOF
-db entry0
-db~1[12]
-EXPECT=<<RUN
-"valid"
+e scr.color=0
```
Comment (radare) [inline test/new/db/archos/linux-x64/dbg_bps]: I do t see the point on setting a breakpoint and not running the code to hit it

Comment (yossizap) [reply test/new/db/archos/linux-x64/dbg_bps]: I am only testing that the fields are properly updated, you will never hit an invalid breakpoint anyways and we have tests for valid breakpoints.

Comment (radare) [reply test/new/db/archos/linux-x64/dbg_bps]: the output matters imho

**Comment by XVilka** [comment]
Good to merge? Please decide

**Comment by ret2libc** [comment]
I don't understand what's the purpose of this PR honestly. As the title says, however, it seems these dbg tests always pass, no matter what. However, even with these changes they always seem to pass, which is strange :/

**Comment by yossizap** [comment]
>I don't understand what's the purpose of this PR honestly. 
>these changes they always seem to pass, which is strange :/

'Always succeeded' as in 'no matter what', the check there was for `db~1[12]` and succeeded even though that field doesn't print `invalid`. With these changes the tests actually fail on wrong output and check the correct fields.

**Comment by XVilka** [comment]
@radare why remove? It should be fine to merge if you agree, otherwise - just close.

**Comment by ret2libc** [comment]
Let's merge this, @yossizap is right. Current tests accept any value no matter what, which is of course wrong. This PR improves the situation.

That said, dbg tests need a lot of love!

**Comment by radare** [comment]
I dont think the change is right because its not doing whst it was suposed to do but i dont wanna block things but i don’t think its improving anything either. Anyway i think i wont care about this until we switch to the new V suite completely

**Pull request merged**